docs/user/multi-tenancy.md

```diff
@@ -80,7 +80,7 @@ In the [Dashboard sidebar](./sidebar.md#client-data) within the Node-RED Editor,
 <img data-zoomable style="max-width: 400px; margin: auto;" src="/images/dashboard-sidebar-clientdata.png" alt="Screenshot of an example 'Client Data' tab"/>
 <em>Screenshot of an example "Client Data" tab</em>
 
-Client data defines information on the user/client interacting with the Dashboard. This data can be appended to every `msg` a node emits, underneath teh `msg._client` object.
+Client data defines information on the user/client interacting with the Dashboard. This data can be appended to every `msg` a node emits, underneath the `msg._client` object.
```

Comment:
@joepavitt I noticed and fixed a typo that isn't related to the docs update in this PR but falls under the "multi-tenancy" section. I came across it while reviewing the link provided in the docs: "_You can read more about this [here](../../user/multi-tenancy.md#configuring-client-data)._"